folium/folium.py

```diff
@@ -62,23 +62,6 @@
 ]
 
 
-class GlobalSwitches(Element):
```

Comment:
this is a potential breaking change for users, so if we want to do this I think we should do it in a separate PR with a motivation why.

Reply:
Ok restored.